nncf/onnx/hardware/fused_patterns.py

```diff
@@ -352,6 +352,18 @@ def create_linear_arithmetic_activations() -> GraphPattern:
     return linear
 
 
+@ONNX_HW_FUSED_PATTERNS.register(HWFusedPatternNames.LINEAR_ARITHMETIC_ACTIVATIONS_ARITHMETIC)
+def create_linear_arithmetic_activations() -> GraphPattern:
+    linear = linear_operations()
```

Comment:
Why do not re-use existing `create_linear_arithmetic_activations`?
Also, the name of the method is a duplicate.

Reply:
Thanks, fixed.